<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameControlsWidget</class>
 <widget class="QWidget" name="FrameControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSpinBox" name="spin_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Frame number</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> f</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_play">
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_loop">
     <property name="toolTip">
      <string>Loop animation</string>
     </property>
     <property name="text">
      <string>Loop</string>
     </property>
     <property name="icon">
      <iconset theme="media-playlist-repeat">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_first">
     <property name="toolTip">
      <string>Go To First Frame</string>
     </property>
     <property name="text">
      <string>Go To First Frame</string>
     </property>
     <property name="icon">
      <iconset theme="go-first">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_prev_kf">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Go To Previous Keyframe</string>
     </property>
     <property name="text">
      <string>Go To Previous Keyframe</string>
     </property>
     <property name="icon">
      <iconset theme="go-previous-skip">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_prev">
     <property name="toolTip">
      <string>Go To Previous Frame</string>
     </property>
     <property name="text">
      <string>Go To Previous Frame</string>
     </property>
     <property name="icon">
      <iconset theme="go-previous">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_next">
     <property name="toolTip">
      <string>Go To Next Frame</string>
     </property>
     <property name="text">
      <string>Go To Next Frame</string>
     </property>
     <property name="icon">
      <iconset theme="go-next">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_next_kf">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Go To Next Keyframe</string>
     </property>
     <property name="text">
      <string>Go To Next Keyframe</string>
     </property>
     <property name="icon">
      <iconset theme="go-next-skip">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="button_last">
     <property name="toolTip">
      <string>Go To Last Frame</string>
     </property>
     <property name="text">
      <string>Go To Last Frame</string>
     </property>
     <property name="icon">
      <iconset theme="go-last">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_play</sender>
   <signal>clicked(bool)</signal>
   <receiver>FrameControlsWidget</receiver>
   <slot>play_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_first</sender>
   <signal>clicked()</signal>
   <receiver>FrameControlsWidget</receiver>
   <slot>go_first()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_prev</sender>
   <signal>clicked()</signal>
   <receiver>FrameControlsWidget</receiver>
   <slot>go_prev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_next</sender>
   <signal>clicked()</signal>
   <receiver>FrameControlsWidget</receiver>
   <slot>go_next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_last</sender>
   <signal>clicked()</signal>
   <receiver>FrameControlsWidget</receiver>
   <slot>go_last()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_frame</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FrameControlsWidget</receiver>
   <slot>commit_time()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>commit_time()</slot>
  <slot>play_toggled(bool)</slot>
  <slot>go_first()</slot>
  <slot>go_prev()</slot>
  <slot>go_next()</slot>
  <slot>go_last()</slot>
 </slots>
</ui>
